features/page_objects/allocation_page.rb

```diff
@@ -3,8 +3,6 @@ class AllocationPage < BasePage
 
   set_url "/case_workers/admin"
 
-  element :notice, "#notice-summary-heading"
```

Comment:
Shouldn't this object be "#notice-summary-heading-id" or is it redundant?

Reply:
thats redundant, feature tests would have broken if not but i checked in any event